packages/components/ContextualMenu/src/ContextualMenuItem.settings.ts

```diff
@@ -11,6 +11,7 @@ export const settings: IComposeSettings<ContextualMenuItemType> = [
     root: {
       accessible: true,
       accessibilityRole: 'menuitem',
+      ...{ acceptsKeyboardFocus: true },
```

Comment:
I have the same question here. Trying to better my understanding 😊

Reply:
Type checking does occur here and I'm trying to "sneak" the prop through